doc/nrf/drivers/gps.rst

```diff
@@ -10,7 +10,9 @@ The API offers functionality to initialize and configure a device, to start and
    :local:
    :depth: 2
 
+.. note::
 
+   The GPS driver is deprecated since v1.7.0, use the `GNSS interface <https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_modem/doc/gnss_interface.html>`_ instead.
```

Comment:
```suggestion
   The GPS driver is deprecated since v1.7.0, use the :ref:`GNSS interface <nrfxlib:gnss_interface>`_ instead.
```

Reply:
Updated the link. I left out the name, because I think it should use the same "GNSS interface" name by default.

Reply:
Good point!